data/templates/legacy-article.mst

```diff
@@ -1,6 +1,9 @@
 {{#css-files}}
 <link rel="stylesheet" type="text/css" href="resource:///com/endlessm/knowledge/data/templates/css/{{{.}}}"/>
 {{/css-files}}
+{{#custom-css-files}}
+<link rel="stylesheet" type="text/css" href="{{{.}}}"/>
+{{/custom-css-files}}
```

Comment:
Yes, you are right! Good catch. I think I overlooked this solution before because I originally wanted the custom file to replace, not augment.